agbenchmark/challenges/define_task_types.py

```diff
@@ -11,7 +11,7 @@ class Info(BaseModel):
 
 
 class Ground(BaseModel):
-    answer: str
+    answer: Optional[str] = None
```

Comment:
we can either remove answer completely, or make sure it is used in every single one. Its a plain text description of a successful challenge -> used mainly for information not for scoring